src/ui/components/atoms/refactor-ghostbutton/Icon.tsx

```diff
@@ -0,0 +1,26 @@
+import circleQuarters from '../../../../assets/icons/circle-quarters.svg?raw';
+
+interface IconProps {
+  size?: number;
+  className?: string;
+  'data-testid'?: string;
+}
+
+function Icon({ size = 24, className, 'data-testid': testId }: IconProps) {
```

Comment:
No, this approach is bad. You need to use the SVG implementation. Read this doc. https://github.com/Ditmar/OpenScience/blob/main/docs/svg-icons-integration.md